k-distribution/tests/regression-new/map-symbolic-tests/failing/assignment-7-spec.k.out

```diff
@@ -0,0 +1,4 @@
+<k>
+  assignmentResult ( X |-> 1
+  ( MAP [ Z <- 3 ] ) )
+</k>
```

Comment:
so, the current output is:
```
<k>
  assignmentResult ( X |-> 1
  ( MAP [ Z <- 2 ] [ Z <- 3 ] ) )
</k>
```

first, it is not unsound, right?

second, i cannot understand why `MAP [ Z <- 2 ] [ Z <- 3 ]` is not simplified to `MAP [ Z <- 3 ]`.  that is, why  the following rule is not applied?
```
rule (MAP:Map [ K1 <- V1 ]) [ K2 <- V2 ] => MAP              [ K1 <- V2 ] requires K1  ==K K2
```

Reply:
This is in the `failing` directory. These tests are not run on CI. They are there to document the fact that this test is broken with current Map reasoning.

Reply:
i'm asking why this failed.  the current java backend with the MAP-SYMBOLIC module should pass this test, i think.